docs/asyncapi.yaml

```diff
@@ -37,170 +37,113 @@ components:
                 maximum: 9999999999
                 example: 1234567890
               apiLink:
-                type: url
                 description: api link for calling file stream of statement file that has been published unto blob
                 example: 'https://myStatementRetreivalApiEndpoint/statement-receiver/statement/v1/FFC_PaymentStatement_SFI_2022_1234567890_2022090615023001.pdf'  
               documentType:
                 type: string
                 enum:
                   - Payment statement
                 description: document type
-                example: 'Payment statement'
+                example: Payment statement
```

Comment:
This looks incomplete, could you add the complete enum list here (I think it's just something like payment schedule or similar)